### ISSUE TITLE
When will the pretrained weights of the SDXL model be released?

### ISSUE THREAD

**Issue by chongxian** [limuloo/MIGC#5]
hello,thanks for your excellent work, will you release the pretrained weights of sdxl in the future?

**Comment by limuloo** [comment]
Thank you very much for recognizing our work.😄

In fact, I have already been training the SDXL version of MIGC. However, the SDXL model has a larger number of parameters compared to the SD1.x models, which will require more computational resources and necessitate further contemplation on additional training techniques. Additionally, the number of Cross-Attention layers in SDXL has been expanded to 70 based on SD1.x, necessitating an ablation study on the integration points of MIGC into SDXL.

In summary, training on SDXL will demand a considerable amount of effort. Once I have finished training, I will share it as soon as possible. Stay tuned.

**Comment by fritol** [comment]
the SD14 is not great in depicting even simple objects 
SDXL needed
but the positional control is pretty good

**Comment by limuloo** [comment]
@fritol 
You can try changing the base model to get better generation quality. 
![image](https://github.com/limuloo/MIGC/assets/132473106/5384afa1-1f54-456b-bbbf-d021b18d2cee)
